eng/packages/http-client-csharp-mgmt/generator/Azure.Generator.Management/src/Providers/OperationMethodProviders/ResourceOperationMethodProvider.cs

```diff
@@ -70,7 +70,7 @@ public ResourceOperationMethodProvider(
             _serviceMethod = method;
             _convenienceMethod = convenienceMethod;
             _isAsync = isAsync;
-            _methodName = methodName;
+            _methodName = methodName ?? convenienceMethod.Signature.Name ?? "Operation";
```

Comment:
could you check if the `Name` property on MethodSignature is nullable?
If this is never nullable, we should not add the last `?? "Operation"`